content/en/docs/contribute/style/style-guide.md

```diff
@@ -368,7 +368,7 @@ Shortcodes inside include statements will break the build. You must insert them
 Use a single newline to separate block-level content like headings, lists, images, code blocks, and others. The exception is second-level headings, where it should be two newlines. Second-level headings follow the first-level (or the title) without any preceding paragraphs or texts. A two line spacing helps visualize the overall structure of content in a code editor better.
 
 ### Headings
-Remember that screen readers are linear output devices, which means that they output items on a page one at a time. If there is a lot of content on your page, applying headings gives the page a structure that allows the user to easily navigate the page or filter topics of interest.
+People accessing this documentation might have trouble seeing, and use a screen reader or other accessibility application. Screen readers in particular are linear output devices: they output items on a page one at a time. If there is a lot of content on a page, you can use headings to give the page an internal structure. A good page structure helps all readers to easily navigate the page or filter topics of interest.
```

Comment:
How about:
People accessing this documentation may use a screen reader or other assistive technology (AT). [Screen readers](https://en.wikipedia.org/wiki/Screen_reader) are linear output devices.

@Rajakavitha1, if you think linking to the AT or screen reader definition is helfpul, there are probably other links that could be used as well.